### PR TITLE
Adds homebrew formula for layer0

### PR DESCRIPTION
**Pull request by nehayward**
**What does this pull request do?**
This adds the homebrew formula for installing layer0.

**How should this be tested?**
Once this is merged in master we can do.
`brew install quintilesims/layer0/layer0`

`(user/repo/formula)`

### CODE REVIEW

**Comment by diemonster** [comment]
This partially addresses https://github.com/quintilesims/layer0/issues/128, sans Windows support (Chocolatey). Is there a better option than Chocolatey at this point? cc @Cryowatt

File: packages/homebrew/layer0.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Layer0 < Formula
+  desc "Framework that helps you deploy to the AWS with minimal fuss"
```
Comment (diemonster) [inline packages/homebrew/layer0.rb]: [nit] typo: 'to the AWS' (unless you're going for an old man vibe)

File: packages/homebrew/layer0.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  homepage "http://layer0.ims.io"
+  url "https://github.com/quintilesims/layer0/archive/v0.10.6.tar.gz"
```
Comment (diemonster) [inline packages/homebrew/layer0.rb]: this is targeting v0.10.X releases, but the PR is targeting `api-refactor` --- we should either preemptively switch to a v0.11 release that doesn't exist yet, or target this PR for `develop`.

Comment (nehayward) [reply packages/homebrew/layer0.rb]: Yup, I'll go ahead and change that.

**Comment by nehayward** [comment]
Yeah I'm curious what the best package managers are for linux and windows.

File: packages/homebrew/layer0.rb (hunk 2 above)
Comment (nehayward) [reply packages/homebrew/layer0.rb]: Actually I think I'll leave it until 11 is read so we can at least test this.

File: packages/homebrew/layer0.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class Layer0 < Formula
+  desc "Framework that helps you deploy to AWS with minimal fuss"
+  homepage "http://layer0.ims.io"
+  url "https://github.com/quintilesims/layer0/archive/v0.10.6.tar.gz"
```
Comment (diemonster) [inline packages/homebrew/layer0.rb]: we should use latest version of Layer0

Comment (diemonster) [reply packages/homebrew/layer0.rb]: btw, is there any way to have these scripts always grab `latest`, or do we always have to manually update them?

Comment (nehayward) [reply packages/homebrew/layer0.rb]: It doesn't look like it. We would probably have to do a script
https://stackoverflow.com/questions/24987542/is-there-a-link-to-github-for-downloading-a-file-in-the-latest-release-of-a-repo

**Pull request merged**